terra_notebook_utils/workflows.py

```diff
@@ -50,6 +51,12 @@ def get_workflow(submission_id: str,
     resp.raise_for_status()
     return resp.json()
 
+def _get(path: str, data: Dict[str, Any]) -> Any:
+    res = jmespath.search(path, data)
+    if res is None:
+        raise TNUCostException(f"'{path}' note found in {json.dumps(data)}")
```

Comment:
If I understand this correctly, it should be:

raise TNUCostException(f"'{path}' **not** found in {json.dumps(data)}")

other than that LGTM

Reply:
Thanks for catching that.